x-pack/plugins/security_solution/public/flyout/left/components/analyze_graph.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React from 'react';
+import { EuiText } from '@elastic/eui';
+import { ANALYSER_GRAPH_TEST_ID } from './test_ids';
+
+export const ANALYZE_GRAPH_ID = 'analyze_graph';
```

Comment:
A nit, but I would unify on `analyze` vs `analyse` (which is what I think is used in the cypress tests)

Reply:
I was trying to figure out what was up...and TIL 😂 

<img width="750" alt="image" src="https://user-images.githubusercontent.com/17211684/223550839-ae7a7cc5-c770-4f64-9832-84123d33f4e3.png">

Reply:
😂 I actually kinda like analyse. Feels fancier 🍷 lol . Thanks for unifying it!